readthedocs/projects/urls/private.py

```diff
@@ -105,10 +106,6 @@
         private.project_redirects_delete,
         name='projects_redirects_delete'),
 
-    url(r'^(?P<project_slug>[-\w]+)/resync_webhook/$',
```

Comment:
I didn't think of this, I'd say it doesn't concern me all that much. With the next PRs, the integrations interface will make more sense, and this URL will be replaced again anyways -- becoming a per-webhook URL instead. So, can probably skip a redirect for now.